pkg/report/report.go

```diff
@@ -552,6 +553,11 @@ func Generate(ctx context.Context, path string, mrs yara.MatchRules, c malconten
 	fr.RiskScore = overallRiskScore
 	fr.RiskLevel = RiskLevels[fr.RiskScore]
 
+	// Ensure that the behaviors are consistently sorted by ID
```

Comment:
We rebuild the `Behaviors` slice when working on overrides, so that's probably why this needs to be done. I can probably do this in `handleOverrides` but this ensures that the sorting happens right before we return the report.